docker/kopia-build/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM golang:1.19.1-alpine3.16 AS builder
+FROM golang:1.19.2-buster AS builder
```

Comment:
Wondering whether this should be `golang:1.19-buster` instead?
I theory, this could make the build not fully reproducible, but OTOH it allows _automatically_ getting the Golang updates.
Let's get @miquella input here.

Reply:
@julio-lopez: I chatted with @bathina2 about this and we believe that we should change to `golang:1.19-buster`, as you suggest.

Although there is a bit of a downside in that it's not explicit which version of 1.19 was being used for a previously built image, it does avoid a lot of the toil in upgrading the version of Go every ~month or so. The Go team is very careful to not release breaking changes in minor versions, so the risk of breaking things is fairly low.